crates/wasmprinter/Cargo.toml

```diff
@@ -14,7 +14,7 @@ Rust converter from the WebAssembly binary format to the text format.
 
 [dependencies]
 anyhow = "1.0"
-wasmparser = { path = '../wasmparser', version = '0.77' }
+wasmparser = { path = '../wasmparser' }
```

Comment:
Since we publish wasmparser we need to keep the dependency version here

Reply:
Ah for some reason I thought that cargo would figure this out manually.